src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <p class="title">Our Ecosystem</p>
+
+        <!-- Subtitle w/ Arrow -->
+        <p class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <p class="prompt-text">EXPLORE</p>
+                <span class="arrow">Arrow</span>
+            </span>
+        </p>
+
+        <!-- Block Links -->
+        <nuxt-link
+            v-for="(brand, index) in brands"
+            :key="index" 
+            class="brand-link"
+            @mouseover.native="set(brand.name)" 
+            @mouseout.native="set(null)"  
+            :to="`/${brand.name}`" tag="span"
+        >
+            <span class="logo">{{brand.name}} </span>{{brand.text}}
+        </nuxt-link>
+
+        <!-- Images absolute -->
+        <wp-image
+            v-for="(image, index) in images"
+            :key="`image-${index}`"
+            class="image"
+            :class="{'is-active': index == activeImage}"
+            :image="image"
+            mode="fullbleed"
+        />
+
+        <!-- Bottom Text -->
+        <p class="end-text">These diverse talent are connected by a creative singularity.</p>
+        <p class="end-text">Together we are <span class="logo">MakeMake</span></p>
+    </div>
+</template>
+
+<script>
+import WpImage from "@/components/global/WpImage"
+import NuxtLink from "@/components/global/NuxtLink"
+
+export default {
+    components: { WpImage, NuxtLink },
+    props: {
+        rpsImages: {
+            type: Array,
+            default: () => []
+        },
+        elasticImages: {
+            type: Array,
+            default: () => []
+        },
+        primaryImages: {
+            type: Array,
+            default: () => []
+        },
+        a52Images: {
+            type: Array,
+            default: () => []
+        },
+        jaxImages: {
+            type: Array,
+            default: () => []
+        },
+        indestructibleImages: {
+            type: Array,
+            default: () => []
+        }
+    },
+    data () {
+        return {
+            brand: null,
+            brands: [
+                { name: 'rock-paper-scissors', text: 'edits films, television, commercials, and music videos.'},
+                { name: 'elastic', text: 'creates animation, motion design, and main titles.'},
+                { name: 'a52', text: 'is a leader in visual effects.'},
+                { name: 'primary', text: 'grades color and manages workflows and telecine.'},
+                { name: 'jax', text: 'markets films, tv shows, and games.'},
+                { name: 'indestructible', text: 'supervises musical direction, composition, and licensing.'},
```

Comment:
This was a good idea! I hadn't though to make the text per company like that. You may was well make these a prop then, so I can pull them from the CMS.

So I imagine you'd want to make the props all be an object per company, like this:

```
rps: {
    slug: `rock-paper-scissors`,
    image: [{}],
    text: "edits films, television, commercials, and music videos."
}
```